deploy/osd-delete-backplane-serviceaccounts/10-delete-backplane-serviceaccounts.CronJob.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: backplane
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: osd-delete-backplane-serviceaccounts
+          containers:
+            - name: osd-delete-backplane-serviceaccounts
+              imagePullPolicy: Always
+              image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+              args:
+                - /bin/bash
+                - -c
+                - |
+                  for sa in $(oc get sa -A -l managed.openshift.io/backplane=true --no-headers | awk '{print $1 ":" $2}');do
+                    name=$(echo $sa | cut -d':' -f2);
+                    namespace=$(echo $sa | cut -d':' -f1);
+                    creation=$(date -d $(oc get sa $name -n $namespace -o jsonpath='{.metadata.creationTimestamp}') '+%s');
+                    now=$(date +%s);
+                    expiryMins=$(oc get sa $name -n $namespace -o jsonpath='{.metadata.annotations.managed\.openshift\.io/backplane-expiry-duration}');
+                    if [[ $(expr $((now-creation)) \> ${expiryMins:=120} \* 60) == 1 ]]; then
+                      oc delete sa $name -n $namespace
+                    fi
```

Comment:
What if I'm using that SA still?  2 hours isn't that long if there's an incident.  I expect I have to log in again.  And if backplane cli were to automate that any active port forwards like Prometheus need restarted.  Backplane is proxying all requests.  Could backplane manage deletion based on the timestamp for last time the SA was used?

Reply:
backplane is not caching the cluster metadata until it's being used, instead of 2 hours, maybe we can set to 12h, or 24h.

Reply:
I like that, 12h may be a good number.  That way it should expire while I'm offline assuming a reasonable work day, even if I created the SA at the beginning or end of the work day.

Reply:
Maybe a INFO output on ocm backplane login about when the SA expires?  it could be simple math based on the creation timestamp but requires both the cli and the job here to be on the same schedule.  small risk of drive, account for in good comments.

Reply:
I'll create a card for this request